Terminal.Gui/Configuration/ConfigurationManager.cs

```diff
@@ -390,7 +390,7 @@ internal static void AddJsonError (string error)
         }
 
         // If value type, just use copy constructor.
-        if (source.GetType ().IsValueType || (source.GetType () == typeof (string)))
+        if (source.GetType ().IsValueType || source.GetType () == typeof (string))
```

Comment:
For the right-hand side of this, which is a type check, it should be the is operator as well, but I'm not sure if ReSharper has a built-in rule for that, specifically. I'll check and report back here.